sdx_pushgateway_sg.tf

```
@@ -11,18 +11,16 @@ resource "aws_security_group" "sdx_pushgateway" {
 }
 
 resource "aws_security_group_rule" "allow_sdx_pushgateway_egress_https" {
-  count             = local.is_management_env ? 0 : 1
   description       = "Allows ECS to pull container from S3"
   type              = "egress"
   protocol          = "tcp"
   from_port         = var.https_port
   to_port           = var.https_port
-  security_group_id = aws_security_group.sdx_pushgateway[local.primary_role_index].id
+  security_group_id = aws_security_group.sdx_pushgateway[0].id
```

don't you need to remove the 0 ? 

```suggestion
  security_group_id = aws_security_group.sdx_pushgateway.id
```

---

the count has to be there because the resources have been deployed to non-mgmt environments already. Without it, it won't work for all environments in this PR. That count can be removed when all environments are the same.